dataline-integrations/singer/postgres/destination/docker/run.sh

```diff
@@ -0,0 +1,38 @@
+#!/bin/bash
```

Comment:
Definitely beyond the scope of this PR, but we should make or find some sort of package that makes this delegation of options to commands easier.